momento-sdk/src/intTest/java/momento/sdk/SimpleCacheClientTest.java

```diff
@@ -190,4 +191,75 @@ public void initializesSdkAndCanHitControlPlaneForUnreachableDataPlane() {
           .hasMessageContaining("server was unable to handle the request");
     }
   }
+
+  @Test
+  public void shouldReturnCacheIncrementedValuesWithStringField() {
+    final String field = "field";
+
+    CacheIncrementResponse incrementResponse =
+        target.increment(cacheName, field, 1, DEFAULT_TTL_SECONDS).join();
+
+    assertThat(incrementResponse).isInstanceOf(CacheIncrementResponse.Success.class);
+    assertThat(((CacheIncrementResponse.Success) incrementResponse).valueNumber()).isEqualTo(1);
+
+    incrementResponse = target.increment(cacheName, field, 50, 1000).join();
```

Comment:
Did you intend to still leave these `1000` ttls in here?

Reply:
Nope, I wanted to use the DEFAULT_TTL. Let me correct this.